src/wabt.cpp

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright 2016-2018 Alex Beregszaszi et al.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if HERA_WABT
```

Comment:
Don't do this. Include the file conditionally in CMake.

Reply:
Can you provide an example how to do it?

Reply:
Thanks!